### PR TITLE
Fix embedding

### PR DESCRIPTION
**Pull request by julianoes**
Just two small fixes.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ include(cmake/tinyxml2.cmake)
 include(cmake/zlib.cmake)
 
 if(NOT MSVC)
+    cmake_policy(SET CMP0058 NEW)
```
Comment (JonasVautherin) [inline CMakeLists.txt]: Do we use the `BYPRODUCT` anywhere?

**Comment by JonasVautherin** [comment]
Why is `embed_files.cmake` not necessary anymore?

**Comment by julianoes** [comment]
> Why is embed_files.cmake not necessary anymore?

This was just a leftover from an earlier try.

File: CMakeLists.txt (hunk 1 above)
Comment (julianoes) [reply CMakeLists.txt]: No idea. It's somewhere in the cpp_rsc cmake code.

**Pull request merged**